### PR TITLE
Tools: DeSET mapping boat default parameters

### PR DESCRIPTION
**Pull request by rmackay9**
This is a non-functional change.  This simply makes parameters available for a new "reference frame" called the "DeSET mapping boat"

### CODE REVIEW

File: Tools/Frame_params/deset-mapping-boat.param [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+RNGFND1_MIN_CM,0
+RNGFND1_TYPE,17
+SERIAL2_BAUD,19
+SERIAL2_PROTOCOL,39
+SERIAL3_BAUD,38
+SERIAL3_PROTOCOL,40
+SERIAL4_BAUD,9
+SERIAL4_PROTOCOL,9
+SERVO_BLH_3DMASK,0
+SERVO_BLH_AUTO,0
```
Comment (peterbarker) [inline Tools/Frame_params/deset-mapping-boat.param:38]: A lot of these are set to their defaults?

**Review by peterbarker — COMMENTED** [review]
Is this a plug-and-float vehicle?  It's a bit interesting specifying serial devices in one of these, isn't it?

File: Tools/Frame_params/deset-mapping-boat.param (hunk 1 above)
Comment (rmackay9) [reply Tools/Frame_params/deset-mapping-boat.param]: nice catch thanks!  that was totally a mistake and I've removed all these param now.

**Comment by rmackay9** [comment]
@peterbarker, Txs for the review.  It's unlikely that someone will be able to exactly buy this frame in the near future (although I am thinking about it) but these parameters are important to getting the torqeedo motor, sonar and AIS system working and the notes on the param file say which serial ports to plug these three devices into.. so I think I'll leave these serial settings in place.  Txs again.

**Pull request merged**